sass/components/_modal.scss

```diff
@@ -250,7 +250,7 @@
         .modal-title {
             overflow: hidden;
             max-width: calc(100% - 80px);
-            color: $bg--gray;
+            color: var(--sidebar-header-text-color);
```

Comment:
Does this also need to be applied to `.name` below as per https://github.com/mattermost/mattermost-webapp/pull/8399/files#diff-efa0d8b3bc74afa1fa4a36331dfa53b1549a04392979e2a2d83fdd914ffc2f4aL583?

Reply:
I couldn't find `.name` used anywhere anymore, added a note to that affect to the description for that commit.